scripts/dasharo-deploy

```diff
@@ -457,14 +859,68 @@ install() {
     fi
     $FLASHROM -p "$PROGRAMMER_BIOS" ${FLASH_CHIP_SELECT} ${FLASHROM_ADD_OPT_REGIONS} -w "$BIOS_UPDATE_FILE" ${_flashrom_extra_args} >> $FLASHROM_LOG_FILE 2>> $ERR_LOG_FILE
     error_check "Failed to install Dasharo firmware"
-
     print_ok "Successfully installed Dasharo firmware"
+
+    return 0
+  fi
+
+  # Must not get here.
+  return 1
+}
+
+install_workflow() {
+# Installation workflow. The installation of firmware is possible only via
+# flashrom, capsules cannot do the installation because they need initial
+# support inside firmware. The workflow steps are:
+# 1) Prepare system for installation (e.g. check connection);
+# 2) Prepare environment for installation (e.g. set all needed vars);
+# 3) Ask user are the changes that will be done ok;
+# 4) Do backup;
+# 5) Do the installation;
+# 6) Do some after-installation routine.
+  sync_clocks
+
+  # Verify that the device is not using battery as a power source:
+  check_if_ac
+  error_check "Firmware update process interrupted on user request."
+
+  # Set all global variables needed for installation:
+  prepare_env install
+
+  # Download and verify firmware:
+  if [ "$HAVE_EC" == "true" ]; then
+    download_ec
+    verify_artifacts ec
+  fi
+  if [ "$CAN_INSTALL_BIOS" == "true" ]; then
+    download_bios
+    verify_artifacts bios
+  fi
+
+  # Ask user for confirmation:
+  display_warning
+
+  backup
+
+  # Deploy EC firmware
+  if [ "$HAVE_EC" == "true" ]; then
+    deploy_ec_firmware install
   fi
 
+  # Deploy BIOS firmware
+  if [ "$CAN_INSTALL_BIOS" == "true" ]; then
+    deploy_firmware install
+  fi
+
+  # Post-installation routine:
   echo -n "Syncing disks... "
   sync
   echo "Done."
-  if [ "$NEED_EC_RESET" = "true" ]; then
+
+  echo "Sending logs..."
```

Comment:
Maybe add this echo to `send_dts_logs` function to not confuse user as it only sends logs if it was enabled.

Reply:
Fixed [here](https://github.com/Dasharo/dts-scripts/compare/ed6c6089caa0e541f15150b15a79a7e26c6ae365..9aa00a04166f712f638acc875e4004c28ece5c32)